app/views/foreman_leapp/job_templates/leapp_install.erb

```diff
@@ -0,0 +1,36 @@
+<%#
+kind: job_template
+name: Install Leaoo
+job_category: Leapp - Install
+description_format: 'Install leapp package'
+provider_type: Ansible
+feature: leapp_upgrade
+model: JobTemplate
+%>
+<%
+  is_rhel = @host.operatingsystem.name == 'RedHat'
+  os_major = @host.operatingsystem.major.to_i
+-%>
+---
+- hosts: all
+  tasks:
+<% if is_rhel &&  os_major == 7 -%>
```

Comment:
Can there be a case when different versions will need different steps? It would be easier to read if we set some variables based on the version and then just had a single set of tasks using those variables